oodeel/methods/odin.py

```diff
@@ -71,11 +71,12 @@ def _score_tensor(self, inputs: TensorType) -> np.ndarray:
         if self.feature_extractor.backend == "torch":
             inputs = inputs.to(self.feature_extractor._device)
         x = self.input_perturbation(inputs)
-        logits = self.feature_extractor.predict(x) / self.temperature
+        _, info = self.feature_extractor.predict(x)
+        logits = info["logits"] / self.temperature
         pred = self.op.softmax(logits)
         pred = self.op.convert_to_numpy(pred)
         scores = -np.max(pred, axis=1)
-        return scores
+        return scores, info
```

Comment:
I think this function should return `scores, logits` since it has no access to the labels, and then info dict would be built in `score` function of the parent class. (same remark for all detectors)